install/operator/.lib.sh

```diff
@@ -149,6 +149,9 @@ FROM golang:1.13.7
 WORKDIR /go/src/${OPERATOR_GO_PACKAGE}
 ENV GO111MODULE=on
 COPY . .
+RUN go mod vendor
+RUN go generate ./pkg/...
+RUN go mod tidy
```

Comment:
Not clear that this should be here either if `go mod vendor` is removed.

Reply:
removed.